modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -280,19 +242,12 @@ public void testAddObjectAction() throws Exception {
 				"url", "https://onafteradd.com"
 			).build());
 
-		try {
-			_addObjectAction(
+		AssertUtils.assertFailure(
+			ObjectActionNameException.class, "Duplicate name " + name,
```

Comment:
What do you thing to add this Assert as the first `AssertUtils` of `ObjectActionNameException.class` to sort by message?